### PR TITLE
Feature navigation desktop

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Navigation Component for Desktop view created
temp-navigation-noucl

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    padding: 1rem 1.5rem;
+    border-bottom: 0.3px solid var(--white);
+    position: sticky;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    /*set margin on divs below nav, nav eats content below it*/
+
+    display: flex;
+    flex-direction: row;
```
Comment (kotalwar-krishna) [inline components/navigation/navigation.css]: display flex already have direction row. no need to write flex-direction row according to me

**Review by kotalwar-krishna — APPROVED** [review]
good to merge

**Review by mariyasada — APPROVED** [review]
looks good to me

File: components/navigation/navigation.css (hunk 1 above)
Comment (abhijitdotsharma) [reply components/navigation/navigation.css]: Yes, It's redundant I'll make the change.

**Pull request merged**